CMakeLists.txt

```diff
@@ -42,9 +43,23 @@ set(C_WARNING_Clang -Wall -Wextra -Werror -Wpedantic
         -Wstrict-prototypes -Wold-style-definition
         -Wno-unused-parameter -Wno-missing-field-initializers -Wno-sign-compare -Wno-language-extension-token)
 
-# Additional compiler-specific flags
-#  -gdwarf-aranges to workaround https://sourceware.org/bugzilla/show_bug.cgi?id=22288 (backtrace line printing with libdw and clang)
 set(C_EXTRA_GNU )
```

Comment:
Why did you put the `if (CMAKE_INTERPROCEDURAL_OPTIMIZATION)` in between the `C_EXTRA_GNU` and `C_EXTRA_Clang` variables? The way I feel it, the C_EXTTRA stuff should be kept together and INTERPROCEDURAL_OPTIMIZATION can be either above it or below it... But this is just a nit.

Reply:
Good catch. I moved it up before the if statement because I *was* setting the `-fno-fat-lto-objects` explicitly but that was later removed since cmake defaults it anyways. I will fix this.